### ISSUE TITLE
[Feature request] Full playstation Move motion support for games like Sports Champions 2

### ISSUE THREAD

**Issue by gamrXerus** [RPCS3/rpcs3#16343]
### Quick summary

Thank you for the preliminary progress on Real Move support on rpcs3. I installed opencv and compiled rpcs3 from source on my system to test real move support and the cursor is very finicky with sports champions 2 in the menus. There is also no motion support in any of the sports games. I have attached a log to help debug the issue. [RPCS3log.tar.gz](https://github.com/user-attachments/files/17881811/RPCS3log.tar.gz) I am willing to send 100USDT (tether) to a developer like @Megamouse if they need to purchase the game to test or buy psmove hardware for reverse engineering if necessary. I think full move support in rpcs3 would greatly enhance the compatibility of games of rpcs3 and add value to the emulator for users like me. I love playing motion games like grand slam tennis, pga tour 12 and wii sports resort on dolphin in 4K with my friends when they come over, so full motion support in rpcs3 would be very useful to me.
Specs
Intel i9 11900K
AMD Radeon RX 7600
Fedora Linux 41 (Kde Plasma)

**Comment by digant73** [comment]
@gamrXerus is this still an issue?

**Comment by Megamouse** [comment]
It's not fully implemented. There's still a bunch of stuff to confirm. But it's a lot better than it used to be.

@gamrXerus if you want to send some money you can do a one time contribution to our patreon. I receive a cut and I already bought a ps eye and two controllers (the old and new model).

**Comment by LucioXerus** [comment]
> It's not fully implemented. There's still a bunch of stuff to confirm. But it's a lot better than it used to be.
> 
> [@gamrXerus](https://github.com/gamrXerus) if you want to send some money you can do a one time contribution to our patreon. I receive a cut and I already bought a ps eye and two controllers (the old and new model).

Yeah the game is not fully supported as yet. I was a patreon subscriber when I was employed. If I get back on my feet soon I have no problem making a one time contribution.

**Comment by home-gihub** [comment]
guys has anything happened?
